File: app/src/main/java/ru/michanic/mymot/UI/Frames/Favourites/FavouritesHomeFragment.java
package ru.michanic.mymot.UI.Frames.Favourites;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.design.widget.TabLayout;
import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentManager;
import android.support.v4.view.ViewPager;
import android.support.v7.widget.Toolbar;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import ru.michanic.mymot.R;
import ru.michanic.mymot.Utils.DataManager;

public class FavouritesHomeFragment extends Fragment {

    private TabLayout tabs;

    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        View rootView = inflater.inflate(R.layout.fragment_catalog_home, null);
        DataManager dataManager = new DataManager();

        /*
        pager = (ViewPager) findViewById(R.id.base_pager);
        fm = getSupportFragmentManager();

        tabs = (TabLayout) rootView.findViewById(R.id.base_tabs);
        //tabs.setupWithViewPager(pager);
        tabs.getTabAt(0).select();

        //--> Adding tabs
        for (int i = 0; i < tabs.getTabCount(); i++) {
            switch (i) {
                case 0:
                    tabs.getTabAt(i).setText("Объявления");
                    break;
                case 1:
                    tabs.getTabAt(i).setText("Модели");
                    break;
                default:
                    tabs.getTabAt(i).setText("Unknown");
                    break;
            }
        }*/

        return rootView;
    }

}

File: app/src/main/java/ru/michanic/mymot/Kotlin/UI/Frames/Favourites/FavouritesHomeFragment.kt
package ru.michanic.mymot.Kotlin.UI.Frames.Favourites

import android.os.Bundle
import com.google.android.material.tabs.TabLayout
import androidx.fragment.app.Fragment
import androidx.viewpager.widget.ViewPager
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import ru.michanic.mymot.Kotlin.UI.Adapters.FavouritesPagerAdapter
import ru.michanic.mymot.R
import ru.michanic.mymot.Kotlin.Utils.DataManager

class FavouritesHomeFragment : Fragment() {
    private var tabs: TabLayout? = null
    override fun onCreateView(
        inflater: LayoutInflater,
        container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        val rootView = inflater.inflate(R.layout.fragment_favourites_home, null)
        val dataManager = DataManager()
        val fragments = arrayOfNulls<Fragment>(1)
        fragments[0] = FavouriteModelsFragment()
        //fragments[1] = FavouriteAdvertsFragment()
        val pager = rootView.findViewById<View>(R.id.base_pager) as ViewPager
        pager.adapter = FavouritesPagerAdapter(childFragmentManager, fragments)
        tabs = rootView.findViewById<View>(R.id.base_tabs) as TabLayout
        tabs?.setupWithViewPager(pager)
        tabs?.getTabAt(0)?.select()

        //--> Adding tabs
        for (i in 0 until (tabs?.tabCount ?: 4)) {
            when (i) {
                0 -> tabs?.getTabAt(i)?.text = "Модели"
                1 -> tabs?.getTabAt(i)?.text = "Объявления"
                else -> tabs?.getTabAt(i)?.text = "Unknown"
            }
        }
        return rootView
    }
}